OverwatchHighlights/HighlightInfo.cs

```diff
@@ -98,7 +85,7 @@ public void Print()
 			Console.WriteLine($"  Skin: {skin}");
 			Console.WriteLine($"  Weapon: {weaponSkin}");
 			Console.WriteLine($"  Intro: {highlightIntro}");
-			Console.WriteLine($"  Unknown Unlock: {unknownUnlock:X16}");
+			Console.WriteLine($"  Type: {highlightType:X16}");
```

Comment:
The `X16` formatter won't work on an enum type, you should be safe to just put `{highlightType}` here.